provider/provider.go

```diff
@@ -90,7 +91,16 @@ func getDefaultPath() string {
 			programFilesDir = path.Join("C:", "Program Files")
 		}
 
-		return path.Join(programFilesDir, "Cyberark Conjur", "Summon", "Providers")
+		dir := path.Join(programFilesDir, "Cyberark Conjur", "Summon", "Providers")
+
+		// enable portable installation with Providers dir next to executable
+		if _, err := os.Stat(dir); os.IsNotExist(err) {
+			exec, _ := os.Executable()
+            execDir := filepath.Dir(exec)
+			dir = path.Join(execDir, "Providers")
+		}
+
+		return dir
```

Comment:
The logic here is a bit inverted to make it easy to read. I think to make it more maintainable you should do something like:
```
if _, err := os.Stat(dir); err == nil {
  return path
}

// Use the directory of the executable otherwise
return filepath.Dir(os.Executable())
```